### PR TITLE
[Model] Add support for nvidia/Llama-3_3-Nemotron-Super-49B-v1

### PR DESCRIPTION
**Pull request by kylehh**
## Motivation

Add support for nvidia/Llama-3_3-Nemotron-Super-49B-v1

## Modifications

Migration nemotron_nas from vllm

## Checklist

- [ ] Format your code according to the [Code Formatting with Pre-Commit](https://docs.sglang.ai/references/contribution_guide.html#code-formatting-with-pre-commit).
- [ ] Add unit tests as outlined in the [Running Unit Tests](https://docs.sglang.ai/references/contribution_guide.html#running-unit-tests-adding-to-ci).
- [ ] Update documentation / docstrings / example tutorials as needed, according to [Writing Documentation](https://docs.sglang.ai/references/contribution_guide.html#writing-documentation-running-docs-ci).
- [ ] Provide throughput / latency benchmark results and accuracy evaluation results as needed, according to [Benchmark and Profiling](https://docs.sglang.ai/references/benchmark_and_profiling.html) and [Accuracy Results](https://docs.sglang.ai/references/accuracy_evaluation.html).
- [ ] For reviewers: If you haven't made any contributions to this PR and are only assisting with merging the main branch, please remove yourself as a co-author when merging the PR.
- [ ] Please feel free to join our Slack channel at https://slack.sglang.ai to discuss your PR.

### CODE REVIEW

File: python/sglang/srt/utils.py [hunk 1]
```diff
@@ -411,6 +425,375 @@ class LayerFn(Protocol):
     def __call__(self, layer_id: int, prefix: str) -> torch.nn.Module: ...
 
 
+class PPMissingLayer(torch.nn.Identity):
```
Comment (merrymercy) [inline python/sglang/srt/utils.py:504]: Do not add these first. Can you drop all of them?

**Review by merrymercy — CHANGES_REQUESTED** [review]

File: python/sglang/srt/utils.py (hunk 1 above)
Comment (kylehh) [reply python/sglang/srt/utils.py:504]: Missinglayer class was migrated from vllm implementation, which was created by Nemotron team. Keep it for code consistent and future proof. Do these cause the failure of CI pipeline?

**Comment by zhyncs** [comment]
@kylehh please rebase. thanks.

**Comment by netanel-haber** [comment]
Hello @merrymercy @zhyncs!
I'm going to attempt integrating this PR into main again, I'll bring it up to speed.
Thanks!
